docs/api/fexpr.rst

```diff
@@ -226,6 +226,12 @@
         * - :meth:`.prod()`
           - Same as :func:`dt.prod()`.
 
+        * - :meth:`.nunique()`
+          - Same as :func:`dt.nunique()`.
+
+        * - :meth:`.countna()`
+          - Same as :func:`dt.countna()`.
```

Comment:
We tend to follow alphabetic order in such lists. May I ask you to also re-arrange `.as_type` and `.prod`?